backend/bitboxbase/handlers/handlers.go

```diff
@@ -433,6 +449,24 @@ func (handlers *Handlers) postRebootBaseHandler(_ *http.Request) (interface{}, e
 	}, nil
 }
 
+func (handlers *Handlers) postUpdateBaseHandler(r *http.Request) (interface{}, error) {
+	handlers.log.Debug("Update Base")
+	payload := struct {
+		Version string `json:"version"`
```

Comment:
If the frontend is providing the version, does that mean the user will have choices which version to update to?

Reply:
Yes. That's how it's designed by @Stadicus, atleast for the MVP.